src/libcollections/lib.rs

```diff
@@ -47,14 +47,14 @@
 #![feature(placement_in)]
 #![feature(placement_new_protocol)]
 #![feature(shared)]
+#![feature(slice_get_slice)]
 #![feature(slice_patterns)]
 #![feature(specialization)]
 #![feature(staged_api)]
-#![feature(step_by)]
 #![feature(trusted_len)]
 #![feature(unicode)]
 #![feature(unique)]
-#![feature(slice_get_slice)]
+#![feature(untagged_unions)]
```

Comment:
Three changes to features are in this file:

1. Removed `step_by`. Old sort used it, but the new one doesn't.
2. Simply moved `slice_get_slice` a few line up to make the feature list sorted alphabetically.
3. Added `untagged_unions`. This feature is used in function `insert_head` above `struct NoDrop<T>`.